src/codeu/chat/server/Controller.java

```diff
@@ -56,6 +59,326 @@ public ConversationHeader newConversation(String title, Uuid owner) {
   }
 
   @Override
+  public Integer toggleRemovedBit(Uuid convo, Uuid user){
+    final ConversationHeader foundConvo = model.conversationById().first(convo);
+    final User foundUser = model.userById().first(user);
+
+    Integer result = 0;
+
+    if(foundConvo != null && foundUser != null) {
+      Integer access = foundConvo.accessControls.computeIfAbsent(user, newAccess -> 0);
+      Integer newAccess;
+
+      // Flag is set to true if the user is removed from a conversation.
+      // Flag stays true once it's set.
+      newAccess = access | ConversationHeader.REMOVED;
```

Comment:
Just do this along with the definition:

`Integer newAccess = access | ConversationHeader.REMOVED;`